packages/core/src/internal/bolt-agent/browser/BoltAgent.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
the file names are created lowercase and separated by `-`. So, this file should be called `bolt-agent.ts` (or move this implementation to be directly in the `index.ts`)